content/en/docs/glossary.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Glossary
+description: A list of Notary Project's terminologies 
+weight: 11
+---
+### Notary 
+Notary is a TUF [(The Update Framework)](https://github.com/theupdateframework/specification) based artifact signing solution. It consists of both [server](https://github.com/notaryproject/notary/blob/master/cmd/notary-server) and [client](https://github.com/notaryproject/notary/blob/master/cmd/notary) components for running and interacting with trusted collections. The client component is referred to as the Notary command-line interface (CLI), which handles tasks such as signing and pushing container images to a registry and updating metadata. The CLI communicates with the registry and the key and metadata management server component.
+### Notation
+Notation is a non-TUF-based artifact signing solution. It refers to the implementation of the Notation CLI and the Notation Go libraries. Notation offers signing and verification workflows for OCI artifacts, enables signature portability across OCI-compliant registries, and facilitates integration with 3rd party key management solutions through a plugin model. The CLI itself is referred to as the "Notation CLI," while the libraries are referrred to as the [Notation-Go](https://github.com/notaryproject/notation-go) library and the [Notation-Core-Go](https://github.com/notaryproject/notation-core-go) library. These libraries can be used independently of the Notation CLI.
+### Notary Project Specifications
```

Comment:
I think we need to explain a dedicated term `Notary Project` in the page.

Reply:
I have included the Notary Project term in the glossary and provided an explanation for it.